### PR TITLE
Add missing fields in charge/credit invoice

### PR DESCRIPTION
**Pull request by everdance**
many fields are missing in charge/credit invoice data type

### CODE REVIEW

**Comment by cristiangraz** [comment]
@everdance Will review today. I took a look at the PR yesterday and the change looks good, but I always verify each of the fields against the documentation, double-check spelling, etc before merging so will have to come back to this to give it a more in-depth look against the docs. Appreciate you adding links to the docs for the webhooks 💯

File: webhooks/credit_invoices.go [hunk 1]
```diff
@@ -24,18 +24,26 @@ type CreditInvoiceNotification struct {
...
+	TotalInCents                  int              `xml:"total_in_cents"`
+	TaxInCents                    int              `xml:"tax_in_cents"`
+	SubtotalInCents               int              `xml:"subtotal_in_cents"`
+	SubTotalBeforeDiscountInCents int              `xml:"subtotal_before_discount_in_cents"`
+	DiscountInCents               int              `xml:"discount_in_cents"`
+	Currency                      string           `xml:"currency"`
+	CreatedAt                     recurly.NullTime `xml:"created_at"`
+	UpdatedAt                     recurly.NullTime `xml:"updated_at"`
+	ClosedAt                      recurly.NullTime `xml:"closed_at"`
+	CustomNotes                   string           `xml:"customer_notes"`
```
Comment (cristiangraz) [inline webhooks/credit_invoices.go]: `/s/CustomNotes/CustomerNotes/`

File: webhooks/charge_invoices.go [hunk 2]
```diff
@@ -24,22 +24,31 @@ type ChargeInvoiceNotification struct {
...
+	SubtotalInCents               int              `xml:"subtotal_in_cents"`
+	SubTotalBeforeDiscountInCents int              `xml:"subtotal_before_discount_in_cents"`
+	Currency                      string           `xml:"currency"`
+	CreatedAt                     recurly.NullTime `xml:"created_at"`
+	UpdatedAt                     recurly.NullTime `xml:"updated_at"`
+	ClosedAt                      recurly.NullTime `xml:"closed_at"`
+	DueOn                         recurly.NullTime `xml:"due_on"`
+	NetTerms                      recurly.NullInt  `xml:"net_terms"`
+	CollectionMethod              string           `xml:"collection_method"`
+	CustomNotes                   string           `xml:"customer_notes"`
```
Comment (cristiangraz) [inline webhooks/charge_invoices.go]: `/s/CustomNotes/CustomerNotes/`

File: webhooks/credit_invoices.go [hunk 3]
```diff
@@ -24,18 +24,26 @@ type CreditInvoiceNotification struct {
 	Invoice CreditInvoice `xml:"invoice"`
 }
 
+// https://dev.recurly.com/page/webhooks#section-credit-invoice-schema
```
Comment (cristiangraz) [inline webhooks/credit_invoices.go]: Comment is really helpful but needs to be moved down one line -- first line of the godoc comment should be the `CreditInvoice` line

File: webhooks/charge_invoices.go [hunk 4]
```diff
@@ -24,22 +24,31 @@ type ChargeInvoiceNotification struct {
 	Invoice ChargeInvoice `xml:"invoice"`
 }
 
+// https://dev.recurly.com/page/webhooks#section-charge-invoice-schema
```
Comment (cristiangraz) [inline webhooks/charge_invoices.go]: Comment is really helpful but needs to be moved down one line -- first line of the godoc comment should be the `ChargeInvoice` line

File: webhooks/credit_invoices.go (hunk 1 above)
Comment (everdance) [reply webhooks/credit_invoices.go]: done

File: webhooks/credit_invoices.go (hunk 3 above)
Comment (everdance) [reply webhooks/credit_invoices.go]: done

File: webhooks/charge_invoices.go (hunk 4 above)
Comment (everdance) [reply webhooks/charge_invoices.go]: done

**Pull request merged**